### DISCUSSION TITLE
SlashCommand

### DISCUSSION THREAD

**Discussion by Esteban-DA-COSTA** [discord-jda/JDA#1568] [Questions and Help] [answered]
How can I use slash command with *JDA version 4.2.0_252* ?
I saw that issue [#1501](https://github.com/DV8FromTheWorld/JDA/pull/1501/files?file-filters%5B%5D=.java) is closed so it's available, no ?
But I can't find class like `ApplicationCommandCreateEvent` on javadoc or intellij

**Comment by MinnDevelopment** [comment, marked as answer]
It is locked, not closed. The PR is a draft and the feature is not available yet.

**Reply by Esteban-DA-COSTA** [reply]
Ok, thanks
Do you have any idea about time before it will be avaible ?

**Reply by LucasDaRY** [reply]
I hope this month... I'm waiting for it by a long time

**Reply by DV8FromTheWorld** [reply]
We are holding off solidifying slash command support as discord has not solidified exactly how they want it to work. It should be solidified soon and we will then look to releasing support in `4.3.0`.

Until then, you are welcome to try out the beta branch that contains the experimental support via jitpack.
Checkout the details in the PR for info on how to use the beta branch: https://github.com/DV8FromTheWorld/JDA/pull/1501
